src/WebJobs.Script/Description/Node/Script/globalInitialization.js

```diff
@@ -8,6 +8,9 @@ return function (context, callback) {{
         context.handleUncaughtException(err.stack);
     }});
 
+    // TEMP HACK: workaround for https://github.com/tjanczuk/edge/issues/325
```

Comment:
the current state of affairs is that ANY calls to process.nextTick currently do not succeed. So with this change we're no worse off. any packages using nexttick currently do not work.